src/AudioManagerImpl.js

```diff
@@ -221,18 +222,25 @@ export default class AudioManagerImpl implements AudioManager {
         });
     }
 
-    playAnnouncement(messageIdSuffix: string, intl: IntlShape, messagePayload: any) {
-        if (this.announcementsEnabled) {
-            if (window.speechSynthesis.speaking || window.speechSynthesis.pending) {
-                window.speechSynthesis.cancel();
-            }
-            const messageId = "Announcement." + messageIdSuffix;
-            const toAnnounce = intl.formatMessage({ id: messageId}, messagePayload);
-            const utterance = new SpeechSynthesisUtterance(toAnnounce);
-            window.speechSynthesis.speak(utterance);
+    playFeedbackAnnouncement(message: string) {
+        if (this.audioFeedbackEnabled) {
```

Comment:
This should also check `audioEnabled`.

Reply:
audioEnabled has a default state value of true in App.js, and we currently have no way to change the audioEnabled value. We should talk about what this state is for as you talked about in one of your comments.